rhoas/kafkas/resource_kafka.go

```diff
@@ -89,6 +90,15 @@ func ResourceKafka() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
+			"acl": {
+				Type:     schema.TypeList,
+				ForceNew: true,
+				Required: true,
```

Comment:
Does this mean that an ACL is required when provisioning a new kafka instance?

Reply:
ah very good, so if the acl is not passed then a list of `blank` acls is created.


so in the case where we want to create a kafka, with no acls just yet, we'ld have to pass in the empty list?
could a check be made in the code to see if the acl list is passed in? if not create the empty acl list in the code as opposed to having the user doing it?

Reply:
done 👍